packages/gatsby-theme-store/src/components/SearchSuggestions/Products/hooks.ts

```diff
@@ -14,6 +15,7 @@ interface Props {
 }
 
 export const useProductsSuggestions = ({ maxItems, term }: Props) => {
+  const regionContext = useRegion()
```

Comment:
if this is a variable to productssuggestions, why not use it as an option to the hook ?

Reply:
If I correctly understood your question, the reason behind it is to keep the context as close as possible to where we use it. Before, I was using it in an outer component and passing the information through props. Breno advised me to put it closed to where it's used, which was the hook itself.